src/rrule.ts

```diff
@@ -13,19 +13,6 @@ import { Cache, CacheKeys } from './cache'
 import { Weekday } from './weekday'
 import { iter } from './iter/index'
 
-interface GetNlp {
-  _nlp: Nlp
-  (): Nlp
-}
-
-const getnlp: GetNlp = function () {
-  // Lazy, runtime import to avoid circular refs.
-  if (!getnlp._nlp) {
-    getnlp._nlp = require('./nlp')
-  }
-  return getnlp._nlp
-} as GetNlp
```

Comment:
The rationale for this IIRC was to make `nlp` an optional dependency - I had in mind to eventually split it out into its own package. But this simplifies things for now, so we should merge it.